lib/service/service.go

```diff
@@ -1860,6 +1878,35 @@ func (process *TeleportProcess) initAuthService() error {
 		})
 	}
 
+	if cfg.AccessGraph.Enabled {
+		log.Debugf("Access Graph integration enabled")
+
+		process.RegisterFunc("access-graph-service", func() error {
+			log.Infof("Starting access graph service")
+
+			accessGraphAddr := cfg.AccessGraph.Addr
+			if accessGraphAddr == "" {
+				return trace.NotFound("access graph endpoint not configured")
+			}
+
+			const accessGraphRetryPeriod = 5 * time.Second
+
+			// TODO(jakule): Very excessive retrying, but we need to make sure that
+			// the access graph is initialized before we start serving requests.
+			for {
+				if err := initializeAndWatchAccessGraph(process.ExitContext(), accessGraphAddr, authServer); err != nil {
```

Comment:
Seems like this does not exit and instead wait forever when receiving SIGQUIT (`pkill -3 teleport`). Same issue as https://github.com/gravitational/teleport/pull/28356 , see that PR for the fix.

Reply:
Thanks. Wasn't aware.